### PR TITLE
Mali replay profiling

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

File: gapis/api/vulkan/replay.go [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ func (a API) Replay(
 			makeReadable.imagesOnly = true
 			optimize = false
 			transforms.Add(NewWaitForPerfetto(req.traceOptions, req.handler, req.buffer))
-			transforms.Add(&profilingLayers{})
+			//transforms.Add(&profilingLayers{})
```
Comment (chrisforbes) [inline gapis/api/vulkan/replay.go]: WIP :)

**Comment by AWoloszyn** [comment]
So, we have to process the mali data differently than other data? This seems to defeat some of the purpose of the standardized data format?

**Comment by chrisforbes** [comment]
I would like to collapse together most of this. There are only some very minor differences (which slices are interesting, how many of them we expect, etc)

File: gapis/api/vulkan/replay.go (hunk 1 above)
Comment (chrisforbes) [reply gapis/api/vulkan/replay.go]: This is now fixed.

**Comment by chrisforbes** [comment]
CI-linux-test failure is unrelated flake

File: gapis/trace/android/mali/profiling_data.go [hunk 2]
```diff
@@ -0,0 +1,281 @@
+// Copyright (C) 2019 Google Inc.
```
Comment (lpy) [inline gapis/trace/android/mali/profiling_data.go]: 2020

**Pull request merged**